molecule/default/verify.yml

```diff
@@ -21,3 +21,13 @@
         tasks_from: test_qdr
       vars:
         qdr_container_name: 'qdrtest'
+
+- name: Dump log
+  hosts: qdrtest
+  tasks:
+    - name: Get qdr log
+      register: qdrlog
+      slurp:
```

Comment:
I learned about a new module today.

Reply:
Yeah unfortunately it doesn't actually run when the tests fail :-( I thought it would since it's on a different host than the failing task, but nope. Still banging my head on this one, but getting closer!